cloudfoundry-client/src/main/java/org/cloudfoundry/client/v3/organizations/OrganizationsV3.java

```diff
@@ -63,4 +63,12 @@
      */
     Mono<ListOrganizationsResponse> list(ListOrganizationsRequest request);
 
+    /**
+     * Makes <a href="http://v3-apidocs.cloudfoundry.org/version/3.77.0/index.html#list-domains-for-an-organization">List Organization Domains</a> request
+     *
+     * @param request the List Organization Domains request
+     * @return the response from the List Organization Domains request
+     */
+    Mono<ListOrganizationDomainsResponse> listDomains(ListOrganizationDomainsRequest request); //TODO needs tests
```

Comment:
Does it still need tests?

Reply:
Good catch! I added test but forgot to remove the TODO 😅